### DISCUSSION TITLE
interval time of reading sensors data

### DISCUSSION THREAD

**Discussion by dohoangminhquan** [theyosh/TerrariumPI#713] [General]
hello,

I'm using HC-SR04 and DS18b20. It looks like TerriariumPi read data from sensors every 1 minute right then it updates the sensor status on dashboard?

Is there a way to set the interval time of reading sensor data so that I can see distance change / temperature faster like we can do with Arduino ?

Thanks

**Comment by theyosh** [comment]
Hi,
the sensors are updated every 30 seconds. The data on the dashboard is also updated every 30 seconds.

But the history values are stored per 60 seconds. Which means 2 measurements per minute are averaged and that average is stored for the graph.

Faster updating is not possible. It will give all kind of issues with timeouts of sensors and the load on the PI.
